components/actionbutton/stories/actionbutton.stories.js

```diff
@@ -81,27 +84,54 @@ export default {
 	},
 };
 
+// ********* DOCS ONLY ********* //
+
 export const Default = ActionButtonGroup.bind({});
-Default.args = {
-	label: "More",
+Default.args = {};
+Default.tags = ["!autodocs"];
+
+/**
+ * Action buttons should always have a label, unless they are only using an icon that is universally understood and accessible. They can have an optional icon, but it should not be used for decoration. Use an icon only when necessary and when it has a strong association with the label text.
+ *
+ * The label can be hidden to create an icon-only action button. If the label is hidden, an icon is required, and the label will appear in a tooltip on hover.
+ */
+
+export const Standard = TreatmentTemplate.bind({});
+Standard.args = Default.args;
+Standard.tags = ["!dev"];
+Standard.parameters = {
+	chromatic: { disableSnapshot: true },
 };
 
-export const Emphasized = ActionButtons.bind({});
+Standard.storyName = "Default";
+
+/**
+ * By default, action buttons are not emphasized. This is optimal for when an action button is not the core part of an interface, such as in application panels, where all the visual components are monochrome in order to direct focus to the content.
+ *
+ * The emphasized action button has a blue background for its selected state in order to provide a visual prominence. This is optimal for when the selection should call attention, such as within a tool bar.
+ */
+
+export const Emphasized = TreatmentTemplate.bind({});
 Emphasized.tags = ["!dev"];
 Emphasized.args = {
-	label: "More",
 	isEmphasized: true,
+	isSelected: true
 };
 Emphasized.parameters = {
 	chromatic: {
 		disableSnapshot: true,
 	},
 };
 
-export const Quiet = ActionButtons.bind({});
+/**
+ * By default, action buttons have a visible background. This style works best in a dense array of controls where the background helps to separate action buttons from the surrounding container, or to give visibility to isolated buttons.
+
+ * Alternatively, quiet action buttons can have no visible background until they’re interacted with. This style works best when a clear layout (vertical stack, table, grid) makes it easy to parse the buttons. Too many quiet components in a small space can be hard to read.
+ */
+
```

Comment:
Just personal preference here: I'd probably remove this "By default, ..." line from the "Quiet" docs. It could go up with the `Standard` story if you really wanted, but then we could make some minimal edits to the quiet paragraph. 


```suggestion
/**
 * Quiet action buttons have no visible background until they’re interacted with. This style works best when a clear layout (vertical stack, table, grid) makes it easy to parse the buttons. Too many quiet components in a small space can be hard to read.
 */
```